websocket-tcp-proxy/src/main.rs

```diff
@@ -1,28 +1,69 @@
 use axum::{
     extract::{
         ws::{Message, WebSocket, WebSocketUpgrade},
-        Path, TypedHeader,
+        Path, Query, State, TypedHeader,
     },
     response::IntoResponse,
     routing::get,
     Router,
 };
-use std::env;
+use bitcoin_hashes::hex::FromHex;
+use bytes::Bytes;
+use futures::lock::Mutex;
+use serde::{de, Deserialize, Deserializer};
+use std::collections::HashMap;
 use std::net::SocketAddr;
+use std::sync::Arc;
+use std::{env, fmt, str::FromStr};
+use tokio::sync::mpsc::{self, Sender};
 use tokio::{
     io::{AsyncReadExt, AsyncWriteExt},
     net::TcpStream,
 };
 use tower_http::trace::{DefaultMakeSpan, TraceLayer};
 
+const PUBKEY_BYTES_LEN: i8 = 33;
```

Comment:
you cast this to `usize` a bunch, should just set this to `usize` here instead of `i8`